src/main/java/org/rumbledb/compiler/TranslationVisitor.java

```diff
@@ -248,39 +245,39 @@ else if (content instanceof JsoniqParser.TypeSwitchExprContext)
     // TODO [EXPRVISITOR] count
     @Override
     public Void visitFlowrExpr(JsoniqParser.FlowrExprContext ctx) {
```

Comment:
The naming here has been catching my attention for quite some time. Since we are touching FLWOR expression and clauses here, I was wondering if it might be the correct time to address it. Since we have a policy of using full names for variables functions etc., could we refactor visitFlowrExpr with visitFlowrExpression.

Additionally, the name Flowr is also a typo that exists also in the grammar file. How would you feel about opening an issue and/or addressing this in a new PR (as autogenerated files will probably show up as quite a lot of line changes).

Reply:
You are correct. However, I think that this means changing the g4 grammar. This should be done in a separate PR in which all we do is change the grammar.

Reply:
Exactly. Will we be addressing this soon or shall we create an issue to tackle it later?

Reply:
I'll open an issue.